### PR TITLE
vendor containers/common@main

### PR DESCRIPTION
**Pull request by vrothberg**
The `IgnorePlatform` options has been removed from the
`LookupImageOptions` in libimage to properly support multi-arch images.
Buildah always set it to true.

Also temporarily remove /usr/share/containers/containers.conf.  c/common
is now throwing warnings when facing unknown keys which unfortunately is
the case with the upstream containers.conf and ultimately breaks CI
since some output checks are failing.

Once an updated containers.conf has been shipped, we can revert the
change.

[NO TESTS NEEDED]

Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
@ashley-cui @rhatdan @containers/podman-maintainers PTAL

**Review by ashley-cui — COMMENTED** [review]
LGTM

File: cmd/buildah/main.go [hunk 1]
```diff
@@ -78,6 +78,8 @@ func init() {
 	}
 
 	cobra.OnInitialize(initConfig)
+	// Disable the implicit `completion` command in cobra.
+	rootCmd.CompletionOptions.DisableDefaultCmd = true
```
Comment (nalind) [inline cmd/buildah/main.go:82]: What was this for?  It wasn't mentioned in the commit log or this PR description.

Comment (vrothberg) [reply cmd/buildah/main.go:82]: I have no idea.  It's too long ago.

I assume something failed or a behavior changed.  Cobra was bumped in this PR as well.

Comment (rhatdan) [reply cmd/buildah/main.go:82]: Best to remove it and see what breaks.

**Pull request merged**